roles/dosomething.sixpack/tasks/main.yaml

```diff
@@ -67,6 +67,7 @@
   become: yes
   template: src={{ item }} dest=/{{ item }} owner=root group=root mode=644
```

Comment:
Is putting the `/` as part of dest var vs including it into the var value itself stylistic choice or best-practice documented somewhere?